src/utils/iconActions.ts

```diff
@@ -44,6 +46,16 @@ export const downloadMultipleSVG = (name, icons, size = 32) => {
   downloadSVGs(_icons, name);
 };
 
+export const downloadIconTypes = (icons: IconSetItem[]) => {
+  const template = `
+export type IconNames = ${icons
+    .map((icon) => `'${icon.properties.name}'`)
+    .join(" | ")}
+`;
+
+  downloadTS("IconNames", template);
+};
+
```

Comment:
The file produced here will be a one-line file, starting from the second line, with each icon name written side by side.

This does not match the sample file output in the PR description.

Reply:
Does it matter if it's not formatted? @aykutkardas

Reply:
It may give an error in many people's linters. Wouldn't it be better to format according to common linter settings?